doc/source/getting_started.rst

```diff
@@ -1,50 +1,51 @@
 Getting started
 ===============
 
-The ``pygranta`` metapackage ensures compatibility between PyGranta packages and
-provides a convenient method for installing packages compatible with a specific
-release of Ansys Granta MI.
+The ``pygranta`` metapackage guarantees mutual compatibility between all PyGranta packages. The packages referenced by a
+``pygranta`` metapackage version are certified to be compatible with each other and with the corresponding Granta MI
+version.
 
-Most packages require access to an installation of Ansys Granta MI. For more
-information on getting a licensed copy of Ansys products, visit the `Ansys website <https://www.ansys.com/>`_.
+.. note::
+   The ``ansys-grantami-bomanalytics`` and ``ansys-grantami-bomanalytics-openapi`` versions included in the metapackage
```

Comment:
Not sure exactly how I would phrase this, it's a tricky one, but perhaps something like "The ``ansys-grantami-bomanalytics`` and ``ansys-grantami-bomanalytics-openapi`` versions included in the metapackage are compatible with the corresponding version of *BoM Analytics Services*, included with Granta MI Restricted Substances and Sustainability Reports.

This is normally the latest version available at the time of the metapackage release, however multiple versions of BoM Analytics Services are compatible with a single version of Granta MI."

This is probably the fiddliest section

Reply:
I've re-worked this section slightly.